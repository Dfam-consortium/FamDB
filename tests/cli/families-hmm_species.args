families
--format
hmm_species
-ad
3
