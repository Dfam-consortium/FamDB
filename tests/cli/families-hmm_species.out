#   <copyright header>
<model1>
MS    TaxId:1 TaxName:root
CC    RepeatMasker Annotations:
CC         Type: 
CC         SubType: 
CC         Species: root
CC         SearchStages: 
CC         BufferStages: 
<model2>
MS    TaxId:2 TaxName:Clade_2
MS    TaxId:3 TaxName:Third_Clade
CC    RepeatMasker Annotations:
CC         Type: 
CC         SubType: 
CC         Species: Clade_2, Third_Clade
CC         SearchStages: 
CC         BufferStages: 
<model3>
MS    TaxId:3 TaxName:Third_Clade
CC    RepeatMasker Annotations:
CC         Type: 
CC         SubType: 
CC         Species: Third_Clade
CC         SearchStages: 30,40
CC         BufferStages: 
